src/scripts/sync_subclassof.py

```diff
@@ -0,0 +1,572 @@
+    if len(df3) == 0:
+        df3 = pd.DataFrame(columns=list(ROBOT_SUBHEADER[0].keys()))
+    # - warning cases
+    #   is.na() checks: only object_mondo_id should ever be missing, but checking both to be safe
+    unmigrated_terms = set(df3[df3['object_mondo_id'].isna()]['object_source_id']).union(
+        set(df3[df3['subject_mondo_id'].isna()]['object_source_id']))
+    if len(unmigrated_terms) >= 200:
+        logging.warning(f'Warning: {len(unmigrated_terms)} unmigrated source terms found in "{ontology_name}"; '
+                        f'higher number than expected.')
+    # - format
+    #   only object_mondo_id should ever be missing, but checking both to be safe
+    df3 = df3[~df3['subject_mondo_id'].isna() & ~df3['object_mondo_id'].isna()].sort_values(common_sort_cols)
+    # - obsolete cases
+    robot_subheader_obs = deepcopy(ROBOT_SUBHEADER)
+    robot_subheader_obs[0]['object_mondo_id'] = 'MONDO:excluded_subClassOf'
+    obsoletes = (df3['subject_mondo_label'].str.startswith('obsolete') |
+                 df3['object_mondo_label'].str.startswith('obsolete'))
+    df3_obs = df3[obsoletes].sort_values(common_sort_cols)
+    # - non-obsolete cases
+    df3 = df3[~obsoletes].sort_values(common_sort_cols)
+    # - save
+    df3_obs = pd.concat([pd.DataFrame(robot_subheader_obs), df3_obs])
+    df3_obs.to_csv(outpath_added_obsolete, sep='\t', index=False)
+    df3 = pd.concat([pd.DataFrame(ROBOT_SUBHEADER), df3])
+    df3.to_csv(outpath_added, sep='\t', index=False)
+
+    # Case 5: SCR is direct in Mondo, and not in the source at all
+    rels_mondo_raw_all: List[RELATIONSHIP] = [x for x in mondo_db.relationships(
+        subjects=list(set(ss_df['subject_id'])), predicates=['rdfs:subClassOf'])]
+    rels_mondo_mondo_all = set(  # filter non-source parents
+        [x for x in rels_mondo_raw_all if any([x[2].startswith(y) for y in MONDO_PREFIX_MAP.keys()])])
+    df5 = pd.DataFrame([{'subject_id': x[0], 'object_id': x[2]} for x in rels_mondo_mondo_all])
+    for pos in ['subject', 'object']:
+        df5[f'{pos}_label'] = df5[f'{pos}_id'].apply(lambda _id: mondo_label_lookup.get(_id, None))
+    src_field = f'in_{source_name}'
+    df5[src_field] = df5.apply(
+        lambda row:
+        # UNSUPPORTED-MISSING: either or both Mondo IDs not mapped to a source
+        'UNSUPPORTED-MISSING' if any(x not in mondo_source_map for x in [row['subject_id'], row['object_id']])
+        # Supported / Unsupported: Whether edge has a direct or indirect SCR in source
+        else 'UNSUPPORTED-SUBCLASS'
+        if (row['subject_id'], 'rdfs:subClassOf', row['object_id']) in in_mondo_direct_not_in_source_mondo_edges
+        else 'SUPPORTED',
+        axis=1)
+    df5 = df5.sort_values([src_field, 'subject_id', 'object_id'])
+    df5.to_csv(outpath_direct_in_mondo_only, sep='\t', index=False)
+
+    # Close out
+    print(f'Completed - Synchronization pipeline: subClassOf - {ontology_name}')
+
+
+def cli():  # todo: #remove-temp-defaults
```

Comment:
## Default settings @matentzn 
Review and decide.

<details><summary>Details</summary>
<p>

Basically there are some toggleable default settings that can be run when running the Python script directly rather than in the makefile.

This is very helpful during development. It helps me quickly run and debug things. And there are some things I can do with this defaults setup that I can't do using a debug config with saved params.

We've discussed this type of thing in the past and you've indicated that you prefer to leave out such defaults code from the final product. That's what this `todo` is. It's an indicator that this stuff should be removed when development is done.

Of course, this feature is too big for a single PR, and there are perhaps it will undergo development for many months, so I'd like to leave the defaults there so that I can toggle their usage when I develop further.

Note that these are off by default. So they will have no effect unless I'm doing local development.

Options:
a. Delete them now; Will slow Joe down when he continues to develop in the future, but he can temporarily add them back in another PR.
b. Leave them for now. Delete at some point in the future when other cases are completed.
c. Simply leave them indefinitely into the future. They are no harm in production, and useful for further development and debugging.

</p>
</details>